### PR TITLE
ref(EventManager): Move Event model creation into separate method

### PR DESCRIPTION
**Pull request by untitaker**
How this could be used in the garbage script:

```python
from sentry.event_manager import *
if not should_process(manager.get_data()):
    hashes = get_hashes_for_event(manager._get_event_model())
```

This will miss a lot of normalization happening in `EventManager.save()`, so this might alert us about grouping changes that don't actually occur. As long as the number is manageable I think this is good enough

### CODE REVIEW

**Review by jan-auer — APPROVED** [review]

File: src/sentry/event_manager.py [hunk 1]
```diff
@@ -705,6 +705,27 @@ def should_filter(self):
     def get_data(self):
         return self._data
 
+    def _get_event_model(self, project_id=None):
```
Comment (tonyo) [inline src/sentry/event_manager.py]: I would be a bit more careful with naming here to minimize any possible confusion: a Django model is basically a class, and here we return a model instance, not really the `Event` model itself.

Comment (untitaker) [reply src/sentry/event_manager.py]: What do you think about `_get_data_as_model` or `_get_event_as_model`?

Comment (tonyo) [reply src/sentry/event_manager.py]: I'd actually prefer something like `_get_event_instance` or possible derivatives.

Comment (untitaker) [reply src/sentry/event_manager.py]: done

**Review by tonyo — APPROVED** [review]

**Review by mitsuhiko — APPROVED** [review]

**Pull request merged**